go-controller/pkg/cni/cni.go

```diff
@@ -180,9 +181,9 @@ func (pr *PodRequest) cmdDel(clientset *ClientSet) (*Response, error) {
 					pr.PodName, pr.nadName, err)
 				return response, nil
 			}
-			vfNetdevName = dpuCD.VfNetdevName
+			netdevName = dpuCD.VfNetdevName
```

Comment:
Should we also change `VfNetdevName` to `NetdevName` in dpu connection details annotation?  maybe it requires changing the API spec first? or is it because this commit is for full mode only and connection details annotation is used with DPU(host) mode.

Reply:
You are right about DPU mode it is used with. I decided to keep it as is to not to change API spec, because no real usage for it yet.